ut/s3_post_multipartobject_action_test.cc

```diff
@@ -153,223 +148,6 @@ TEST_F(S3PostMultipartObjectTest, VaidateEmptyTags) {
                action_under_test->get_s3_error_code().c_str());
 }
 
-TEST_F(S3PostMultipartObjectTest, UploadInProgress) {
```

Comment:
Can we have all these below TCs as is, but in a disabled state, say, by using #if 0 pragma. Later, when functionality is complete, we can re-enable them. You may also want to mark a label for such work in code, so that one can search for such term in the entire codebase and work on those items later. We may label it as "TODO:s3fault"

Reply:
All Uts succeding with this changes, these Uts not needed as we have removed some functions